src/format.rs

```diff
@@ -791,8 +791,12 @@ mod test {
     use super::EncryptionFormat;
 
     #[test]
+    #[allow(clippy::eq_op)]
```

Comment:
Drop the allow at line 794, the allow inside the body of the function is enough.

Reply:
Done; sorry about the mess.